WTWSMS/history/characters/WtWSMS_zusk_characters.txt

```diff
@@ -1072,14 +1072,28 @@
 	trait="zealous"
 	trait="strong"
 	father = 7255
-	420.1.1={
+	405.1.1={
 		birth = yes
 	}
-	462.1.1={
+	454.1.1={
 		death = yes
 	}
 }
 
+7259 = {
+	name="Muiredach"
+	dynasty=100005
+	religion="celtic_pagan"
+	culture="irish"
+	father = 7257
+	445.1.1={
+		birth="445.1.1"
+	}
+	489.1.1={
+		death="489.1.1"
+	}
+}
+
```

Comment:
Why move this character away from irish.txt?

Reply:
I was getting confused by having to go from one to another checking the parents and children and ended up moving that one. Can move him back though

Reply:
Preferably all would be in irish.txt, I would like to remove the separate one, although not something for a branch which modifies history already.

Reply:
Should I add all Irish cultured characters to Irish.txt then?

Reply:
Not in the branch since it might hide potential issues, but in the future once it has been merged, yes.

Reply:
In general avoiding to move characters is better since any move will make it difficult to see potentially undesireable modifications.